src/app/layouts/base-layout/components/header/nav-item/nav-item.tsx

```diff
@@ -17,11 +17,39 @@ type NavItemProps = {
 export const NavItem = ({ label, href, dropdownInner }: NavItemProps) => {
   const [isDropdownOpen, setDropdownOpen] = useState(false);
 
+  const dropdownToggleRef = useRef<HTMLButtonElement>(null);
+
   const onClose = () => setDropdownOpen(false);
   const onOpen = () => setDropdownOpen(true);
 
+  const location = useLocation();
+
+  const handleConfirmKeyPress = (e: React.KeyboardEvent<HTMLButtonElement>) => {
```

Comment:
In my opinion using `React` global is outdated and should not be used anymore

<img width="480" alt="Screenshot 2024-09-02 at 16 06 23" src="https://github.com/user-attachments/assets/47dacf9f-c2a2-4686-a8ae-ea7a3b072c88">

```suggestion
  const handleConfirmKeyPress = (e: KeyboardEvent<HTMLButtonElement>) => {
```

Reply:
done